front/app/modules/commercial/customizable_navbar/admin/components/PageFormWithNavbarNameField.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
```

Comment:
Similar to the `PageForm` component, but including a navbar item name field. Used for pages that are in the navbar